cmd/ido2icingadb/README.md

```diff
@@ -0,0 +1,144 @@
+# Icinga DB Migration
+
+![Icinga Logo]
+
+#### Table of Contents
+
+- [About](#about)
+- [License](#license)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Support](#support)
+- [Contributing](#contributing)
+
+## About
+
+This commandline tool migrates history data from [IDO] to [Icinga DB].
+Or, more precisely: from the IDO SQL database to the Icinga DB one.
+
+!!! info
+
+    Everything else is already populated by Icinga DB itself.
+    Only the past history data of existing IDO setups
+    isn't known to Icinga DB without migration from IDO.
+
+## License
+
+This tool and its documentation are licensed under the terms
+of the GNU General Public License Version 2,
+you will find a copy of this license in the [LICENSE] file.
+
+## Installation
+
+1. Add the official release repository as described
+   in the [installation chapter] of the Icinga DB documentation.
+2. Install the package `icingadb-migration`.
+
+## Usage
+
+### Icinga DB
+
+1. Make sure Icinga DB is up, running and writing to its database.
+2. Optionally disable Icinga 2's IDO feature.
+
+!!! warning
+
+    Migration will cause duplicate Icinga DB events
+    for the period both IDO and Icinga DB are active.
+    Disable the IDO feature -- the sooner, the better!
+    Or read on while not disabling it yet.
+    There is a way to avoid duplicate events.
+
+### Configuration file
+
+Create a YAML file like this somewhere:
+
+```yaml
+icinga2:
+   # Content of /var/lib/icinga2/icingadb.env
+   env: "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+   # Name of the main Icinga 2 endpoint writing to IDO
+   endpoint: master-1
+# IDO database
+ido:
+   type: pgsql
+   host: 192.0.2.1
+   port: 5432
+   database: ido
+   user: icingadb_migration
+   password: a23456
+# Icinga DB database
+icingadb:
+   type: mysql
+   host: 2001:db8::1
+   port: 3306
+   database: icingadb
+   user: icingadb_migration
+   password: fd3e4aa3e0d5c833dd17f5c157f9a9d2
```

Comment:
I think most users will just use the regular IDO and Icing DB credentials here. So might be a good idea to use the same database names and users as we do in the setup instructions.